cli/hzn.go

```diff
@@ -105,8 +107,8 @@ Environment Variables:
 	archCmd := app.Command("architecture", msgPrinter.Sprintf("Show the architecture of this machine (as defined by Horizon and golang)."))
 
 	exchangeCmd := app.Command("exchange", msgPrinter.Sprintf("List and manage Horizon Exchange resources."))
-	exOrg := exchangeCmd.Flag("org", msgPrinter.Sprintf("The Horizon exchange organization ID. If not specified, HZN_ORG_ID will be used as a default.")).Short('o').String()
-	exUserPw := exchangeCmd.Flag("user-pw", msgPrinter.Sprintf("Horizon Exchange user credentials to query and create exchange resources. If not specified, HZN_EXCHANGE_USER_AUTH will be used as a default. If you don't prepend it with the user's org, it will automatically be prepended with the -o value. As an alternative to using -o, you can set HZN_ORG_ID with the Horizon exchange organization ID")).Short('u').PlaceHolder("USER:PW").String()
+	exOrg := exchangeCmd.Flag("org", msgPrinter.Sprintf("The Horizon Exchange organization ID. If not specified, HZN_ORG_ID will be used as a default.")).Short('o').String()
```

Comment:
I think we have to leave the case alone. Please back out those changes.

Reply:
I have backed out all the case changes, only the create and update command changes remain